docs/website/docs/dlt-ecosystem/destinations/athena.md

```diff
@@ -77,83 +76,78 @@ profile_name="dlt-ci-user"
-scanning your bucket and reading all relevant parquet files in there.
+Data loading happens by storing parquet files in an S3 bucket and defining a schema on Athena. If you query data via SQL queries on Athena, the returned data is read by scanning your bucket and reading all relevant parquet files in there.
 
 `dlt` internal tables are saved as Iceberg tables.
 
 ### Data types
-Athena tables store timestamps with millisecond precision and with that precision we generate parquet files. Mind that Iceberg tables have microsecond precision.
+Athena tables store timestamps with millisecond precision, and with that precision, we generate parquet files. Keep in mind that Iceberg tables have microsecond precision.
 
-Athena does not support JSON fields so JSON is stored as string.
+Athena does not support JSON fields, so JSON is stored as a string.
 
 > ❗**Athena does not support TIME columns in parquet files**. `dlt` will fail such jobs permanently. Convert `datetime.time` objects to `str` or `datetime.datetime` to load them.
 
 ### Naming Convention
-We follow our snake_case name convention. Mind the following:
-* DDL use HIVE escaping with ``````
+We follow our snake_case name convention. Keep the following in mind:
+* DDL uses HIVE escaping with ``````
 * Other queries use PRESTO and regular SQL escaping.
 
 ## Staging support
 
-Using a staging destination is mandatory when using the athena destination. If you do not set staging to `filesystem`, dlt will automatically do this for you.
+Using a staging destination is mandatory when using the Athena destination. If you do not set staging to `filesystem`, `dlt` will automatically do this for you.
 
 If you decide to change the [filename layout](./filesystem#data-loading) from the default value, keep the following in mind so that Athena can reliably build your tables:
- - You need to provide the `{table_name}` placeholder and this placeholder needs to be followed by a forward slash
- - You need to provide the `{file_id}` placeholder and it needs to be somewhere after the `{table_name}` placeholder.
- - {table_name} must be the first placeholder in the layout.
+ - You need to provide the `{table_name}` placeholder, and this placeholder needs to be followed by a forward slash.
+ - You need to provide the `{file_id}` placeholder, and it needs to be somewhere after the `{table_name}` placeholder.
+ - `{table_name}` must be the first placeholder in the layout.
 
 
 ## Additional destination options
 
-### iceberg data tables
-You can save your tables as iceberg tables to athena. This will enable you to for example delete data from them later if you need to. To switch a resouce to the iceberg table-format,
-supply the table_format argument like this:
+### Iceberg data tables
+You can save your tables as iceberg tables to Athena. This will enable you, for example, to delete data from them later if you need to. To switch a resource to the iceberg table format, supply the table_format argument like this:
```

Comment:
```suggestion
You can save your tables as Iceberg tables to Athena. This will enable you, for example, to delete data from them later if you need to. To switch a resource to the iceberg table format, supply the table_format argument like this:
```